File: Presentacion 1 Portafolio/Cesfam 25-04-2017/Cesfam/clases/Cl_medicamento.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace clases
{
   public class Cl_medicamento
    {

        private int codMedicamento;

        public int CodMedicamento
        {
            get { return codMedicamento; }
            set { codMedicamento = value; }
        }

        private string nombre;

        public string Nombre
        {
            get { return nombre; }
            set { nombre = value; }
        }

        private string componentes;

        public string Componentes
        {
            get { return componentes; }
            set { componentes = value; }
        }



        private string tipo;

        public string Tipo
        {
            get { return tipo; }
            set { tipo = value; }
        }

        private int gramaje;

        public int Gramaje
        {
            get { return gramaje; }
            set { gramaje = value; }
        }

        private DateTime fecVencimiento;

        public DateTime FecNacimiento
        {
            get { return fecVencimiento; }
            set { fecVencimiento = value; }
        }

        private int cantidad;

        public int Cantidad
        {
            get { return cantidad; }
            set { cantidad = value; }
        }



        
    }
}

File: Java/Cesfam2/src/java/clases/Cl_Medicamento.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package clases;

/**
 *
 * @author Coke
 */
public class Cl_Medicamento {
    int codMedicamento;
    String nombre;
    String componentes;
    String tipo;
    int gramaje;

    public Cl_Medicamento() {
    }

    public Cl_Medicamento(int codMedicamento, String nombre, String componentes, String tipo, int gramaje) {
        this.codMedicamento = codMedicamento;
        this.nombre = nombre;
        this.componentes = componentes;
        this.tipo = tipo;
        this.gramaje = gramaje;
    }

    public int getCodMedicamento() {
        return codMedicamento;
    }

    public void setCodMedicamento(int codMedicamento) {
        this.codMedicamento = codMedicamento;
    }

    public String getNombre() {
        return nombre;
    }

    public void setNombre(String nombre) {
        this.nombre = nombre;
    }

    public String getComponentes() {
        return componentes;
    }

    public void setComponentes(String componentes) {
        this.componentes = componentes;
    }

    public String getTipo() {
        return tipo;
    }

    public void setTipo(String tipo) {
        this.tipo = tipo;
    }

    public int getGramaje() {
        return gramaje;
    }

    public void setGramaje(int gramaje) {
        this.gramaje = gramaje;
    }

    @Override
    public String toString() {
        return "Cl_Medicamento{" + "codMedicamento=" + codMedicamento + ", nombre=" + nombre + ", componentes=" + componentes + ", tipo=" + tipo + ", gramaje=" + gramaje + '}';
    }
    
    
}
